tutoroars/templates/oars/jobs/init/superset-api-dashboard.sh

```diff
@@ -1,5 +1,25 @@
 #!/usr/bin/env bash
 set -e
 
-pip install -r /app/oars/pythonpath/requirements.txt
-python /app/oars/pythonpath/superset-dashboard.py && echo "Success"
+#
+# Always install local overrides first
+#
+/usr/bin/env bash /app/docker/docker-bootstrap.sh
+
+/usr/bin/env bash /app/docker/docker-bootstrap.sh
```

Comment:
Looks like you're bootstrapping twice here?

Reply:
ups